source/elements/oneDPL/source/parallel_api/execution_policies.rst

```diff
@@ -151,13 +160,23 @@ Return the queue the policy is associated with.
 
 Allow implicit conversion of the policy to a ``sycl::queue`` object.
 
-Predefined Device Policy
-^^^^^^^^^^^^^^^^^^^^^^^^
+Predefined Device Policies
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 ``dpcpp_default`` is a predefined execution policy object to run algorithms on the default SYCL device.
 It is a global immutable (``const``) instance of type ``device_policy<>``.
 [*Note*: ``dpcpp_default`` can be copied but cannot be moved. -- *end note*]
```

Comment:
Would we want to deprecate this policy object as well to convince people to switch to the new ones, or leave it as is?

Reply:
Deprecate or redefine to be an instance of `dpdefault`?

Reply:
If I understand correctly, beyond just the name shortening, I think the other advantage here is that the new policy templates are on-demand usage and cannot cause problems for broken SYCL environments on startup with eager global queue creation. If we allow an easy path for users to continue using `dpcpp_default` as a instance of the new template, then we perpetuate both of the things we are aiming to improve.  

I don't think we can just remove it, but we can deprecate it for eventual removal of the requirement to have some lazy queue creation system in oneDPL implementations. (https://github.com/uxlfoundation/oneDPL/pull/1652)